project-koku/koku
# [COST-3208] add lineitemdescription to aws line items daily

## Jira Ticket

[COST-3208](https://issues.redhat.com/browse/COST-3208)

## Description

- Adds the legal entity column as required for AWS
- Adds legal entity to daily group by columns
- Adds migration to drop aws daily Trino table (it will be recreated with the new column on next data load)

Testing Instructions:
- Run standard test customer data flow on main
- Switch to this branch
- Run migrate_trino.py script to drop table
- Run show tables; in trino to verify table is gone
- Add and process a new source.
- Check that table exists with new column.


## Notes

...
